test/suites/clustering.sh

```diff
@@ -2514,6 +2514,15 @@ test_clustering_fan() {
   LXD_DIR="${LXD_ONE_DIR}" lxc exec c1 -- ping -c2 -W5 "${IP_C2}"
   LXD_DIR="${LXD_ONE_DIR}" lxc exec c2 -- ping -c2 -W5 "${IP_C1}"
 
+  echo "Check that the DHCP leases are cleaned up post-migration"
```

Comment:
Could you copy the exact same `grep` (without the surrounding `if`) here to ascertain that the pattern we look for is there when it should be and goes away when it should.